.circleci/config/commands/setup-go.yml

```diff
@@ -24,6 +24,7 @@ steps:
         rm -f "go${GO_VERSION}.linux-amd64.tar.gz"
         GOPATH="/home/circleci/go"
         mkdir $GOPATH 2>/dev/null || { sudo mkdir $GOPATH && sudo chmod 777 $GOPATH; }
+        mkdir $GOPATH/bin 2>/dev/null || { sudo mkdir $GOPATH/bin && sudo chmod 777 $GOPATH/bin; }
```

Comment:
I added this step because CI failed with:
`cp: cannot create regular file '/home/circleci/go/bin/': Not a directory`